### ISSUE TITLE
Add output files description table to wiki

### ISSUE THREAD

**Issue by robinwassen** [cefsharp/CefSharp#1122]
A output files description table on the wiki would be helpful for us who build production apps and want to keep the install of them as clean as possible.

Something similar to:

Filename | Required | Description
-------------|-------------|-----------------
CefSharp.BrowserSubprocess.exe|Yes|Browser subprocess executable
CefSharp.BrowserSubprocess.Core.dll|Yes|Browser subprocess stuff
CefSharp.Core.dll|Yes|Core library for CefSharp
CefSharp.dll|Yes|Core library for CefSharp
CefSharp.Wpf.dll|Yes|WPF controls for CefSharp
d3dcompiler_43.dll|Yes|cef library
d3dcompiler_47.dll|Yes|cef library
ffmpegsumo.dll|Yes|cef library
libcef.dll|Yes|cef library
libEGL.dll|Yes|cef library
libGLESv2.dll|Yes|cef library
pdf.dll|Yes|cef library
icudtl.dll|Yes|cef library
cef.pak|Don't know|Don't know
cef_100_percent.pak|Don't know|Don't know
cef_200_percent.pak|Don't know|Don't know
devtools_resources.pak|Don't know|Don't know
locales\\*.pak|Don't know|Don't know

**Comment by robinwassen** [comment]
Related issue: https://github.com/cefsharp/CefSharp/issues/527

**Comment by robinwassen** [comment]
Related README: https://bitbucket.org/chromiumembedded/cef/src/aefb5ccce879f308f0bcc3ac719c86defe2f9715/tools/distrib/win/README.redistrib.txt?at=master

**Comment by jornh** [comment]
Good first stab at it :+1: The wiki is editable by any GitHub user and contributions are welcome.

Related FAQ entry (with more info on some of the deps): https://github.com/cefsharp/CefSharp/wiki/Frequently-asked-questions#Runtime_dependencies

As you already found either through Google or the link in the FAQ entry most dependencies are brought in from the upstream CEF and Chromium projects via the `cef.redist` NuGet package. This one probably evolves over time. E.g. - optional - Flash support via `pepper` will soon land. As the optional `pdf.dll` previously did.

Also see the https://github.com/cefsharp/CefSharp/blob/master/CefSharp/DependencyChecker.cs which tries to inform you about this - if you: **a)** see it, **b)** enable it ... :smile: 

To sum up: I guess the problem will be keeping this info easy to discover, keep maintained, not too redundant AND in sync with the codebase.

Thoughts?

**Comment by robinwassen** [comment]
I agree that it might be hard to keep it in sync, but in order to make it a bit better I did the following:

1. Created a [wiki page](https://github.com/cefsharp/CefSharp/wiki/Output-files-description-table-(Redistribution)) dedicated to the output files, with a table describing each file and stating that the reader should go to either DependencyChecker.cs or the cef redistribution readme if the information is out of date.
2. Removed the inline description from the FAQ # 4 and instead linked to that page
3. Added an FAQ entry for "What files do I need to include when I redistribute an application that uses CefSharp?"

I think this can be considered solved for now.

**Comment by amaitland** [comment]
There's one of the `.pak` files you require otherwise the scrollbars will turn red. So it's not really optional, also you can't disable pak loading these days for the same reason. 

From memory it's `cef_100_percent.pak`

**Comment by amaitland** [comment]
I think it's also important to include information about version. There will potentially need to be multiple tables or version specific notes.

`pdf.dll` for example will be removed for version `2357` as it's compiled into `cef`

**Comment by robinwassen** [comment]
I added a note about the scrollbars at least: https://github.com/cefsharp/CefSharp/wiki/Output-files-description-table-(Redistribution)
